install.sh

```diff
@@ -82,15 +89,19 @@ else
 fi
 
 # Go over board to avoid "rm -rf /"; e.g. APP_PATH is set above, testing anyway.
-if [[ -z "$APP_PATH" || -z "${APP_DIR}" || "$INSTALL_DIR" -eq "/" ]]; then
+if [[ -z "$APP_PATH" || -z "${APP_DIR}" || "$INSTALL_DIR" = "/" ]]; then
   echo "Something is incorrect about the install directory. Exiting."
   exit -1
 fi
 # Yes, this is redundant with the above. User safety is top priority.
-if [[ -n "$APP_PATH" && -n "${APP_DIR}" && "$INSTALL_DIR" -ne "/" ]]; then
-  rm -rf "$INSTALL_DIR"
+# Only continues to delete if directory exists and is not root '/'
+if [[ -n "$APP_PATH" && -n "${APP_DIR}" && ! "$INSTALL_DIR" = "/" ]]; then
```

Comment:
Thanks! That note about -eq is good to know. After reading up on it more it looks like this is now POSIX compliant, which is cool. Though the = in a conditional stings my c++/python mind :)
Can != be used here and == used on line 92? (I admit, just for my own sanity). wdyt? (and is the shell checker cool with != and ==)

Reply:
I updated to `==` and `!=` as it is absolutely more common across languages and therefore less confusing.  I think it's good to have readable code, and especially since this passes the `shellcheck` linter.